opticlust/clust.py

```diff
@@ -22,7 +22,7 @@ def clustering(
     :param cluster_kwargs: kwargs passed on the cluster function
     :return: columns: list of column names generated in adata.obs
     """
-    if "pca" not in adata.uns.keys():
+    if "pca" not in adata.uns: # removed .keys() due to ruff linter, check
```

Comment:
no need for the comment :)

Reply:
Alright, will just remove the comment :)